discuss/email_servers.rst

```diff
@@ -35,6 +34,13 @@ Here is a typical configuration for a G Suite server.
 
 Then set your email domain name in the General Settings.
 
+.. note:: If you get a *[AUTHENTICATIONFAILED] Invalid credentials (Failure)* warning when you *Test
```

Comment:
Nitpicking: You could take the opportunity to return to a new line after the `::` for better code readability.